app/src/main/java/com/cornellappdev/volume/adapters/HomeArticlesAdapter.kt

```diff
@@ -0,0 +1,60 @@
+package com.cornellappdev.volume.adapters
+
+import android.content.Intent
+import android.os.Build
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.RequiresApi
+import androidx.recyclerview.widget.RecyclerView
+import com.cornellappdev.volume.MainActivity
+import com.cornellappdev.volume.databinding.ItemHomeArticleBinding
+import com.cornellappdev.volume.models.Article
+import com.squareup.picasso.Picasso
+
+class HomeArticlesAdapter(private val articles: MutableList<Article>) :
+        RecyclerView.Adapter<HomeArticlesAdapter.HomeArticleVH>() {
+
+    class HomeArticleVH(val binding: ItemHomeArticleBinding) : RecyclerView.ViewHolder(binding.root)
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): HomeArticleVH {
+        val binding = ItemHomeArticleBinding
+                .inflate(LayoutInflater.from(parent.context), parent, false)
+        return HomeArticleVH(binding)
+    }
+
+    @RequiresApi(Build.VERSION_CODES.O)
+    override fun onBindViewHolder(holder: HomeArticleVH, position: Int) {
+        val currentItem = articles[position]
+        holder.binding.tvArticleTitle.text = currentItem.title
+        Article.applyNSFWFilter(currentItem, holder.binding.tvArticleTitle)
+        if (!currentItem.imageURL.isNullOrBlank()) {
+            holder.binding.ivArticleImage.visibility = View.VISIBLE
+            Picasso.get().load(currentItem.imageURL).into(holder.binding.ivArticleImage)
+        }
+        Article.setCorrectDateText(currentItem, holder.binding.tvTimePosted)
+        holder.binding.tvShoutoutCount.text =
+                currentItem.shoutouts?.toInt().toString() + " shout-outs"
```

Comment:
Shout-outs here compared to " shoutouts" before. Is this intentional?

Reply:
had to double check designs, i think shout-outs is intentional